### ISSUE TITLE
train softmax weights of a pretrained transformer model. 

### ISSUE THREAD

**Issue by nimaous** [tensorflow/tensor2tensor#919]
### Description

I already have trained a transorfmer model. Now I want to use this pretrained model with some new example to train only softmax weights of transformer decoder. any help how can I do this? 
1- how can I include my pretrained model
2- how can I reach softmax weights
3- how can I only train softmax wights with other variables remain fixed.  
### Environment information

```
OS: <Ubuntu>

$ pip freeze | grep tensor
tensor2tensor==1.5.7
tensorboard==1.6.0
tensorflow-gpu==1.6.0
tensorflow-tensorboard==1.5.0

$ python -V
Python 3.4.5```
